scripts/test-certificate-request.py

```diff
@@ -100,25 +103,35 @@ def main():
                   ClientCertificateType.ecdsa_sign]
 
     argv = sys.argv[1:]
-    opts, args = getopt.getopt(argv, "h:p:e:x:X:s:k:c:T:dMn:", ["help", "ems"])
+    opts, args = getopt.getopt(argv, "h:p:e:x:X:s:k:c:T:n:dC:M", ["help", "ems"])
     for opt, arg in opts:
         if opt == '-h':
             hostname = arg
         elif opt == '-p':
             port = int(arg)
         elif opt == '-e':
             run_exclude.add(arg)
-        elif opt == '-n':
-            num_limit = int(arg)
-        elif opt == '-d':
-            dhe = True
         elif opt == '-x':
             expected_failures[arg] = None
             last_exp_tmp = str(arg)
         elif opt == '-X':
             if not last_exp_tmp:
                 raise ValueError("-x has to be specified before -X")
             expected_failures[last_exp_tmp] = str(arg)
+        elif opt == '-n':
+            num_limit = int(arg)
+        elif opt == '-d':
+            dhe = True
+        elif opt == '-C':
+            if arg[:2] == '0x':
+                ciphers = [int(arg, 16)]
+            else:
+                try:
+                    ciphers = [getattr(CipherSuite, arg)]
+                except AttributeError:
+                    ciphers = [int(arg)]
+        elif opt == '-M' or opt == '--ems':
+            ems = True
```

Comment:
This case is still handled below in line 153, you should drop it there if you're moving it up.

Reply:
thanks! yes, missed removal of the duplicate case, fixed now